en/docs/choreo-samples/quick-deploy-a-sample.md

```diff
@@ -0,0 +1,33 @@
+# Quick Deploy a Sample
+
+Choreo provides a quick deploy option for samples, allowing you to deploy a sample with a single click. This capability lets you quickly explore the sample and its functionalities.
```

Comment:
```suggestion
Choreo provides a quick deploy option for samples, allowing you to deploy a sample with a single click. This capability lets you quickly explore the sample and Choero functionalities.
```

Reply:
Shall we go with "Choreo provides a quick deploy option for samples, allowing you to deploy a sample with a single click. This capability lets you quickly explore the sample." 
It is a bit confusing to say Choreo functionalities here because each sample demonstrates a specific use case that'll include just a small subset of Choreo capabilities.

Reply:
+1